src/System.Net.Primitives/tests/PalTests/System.Net.Primitives.Linux.Pal.Tests.csproj

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@pgavlin At this point we have two classes of tests: 
* UnitTests: these require access to internals, use fakes, run fast and must be agnostic to the machine they run on. They are not allowed to make any kind of system calls (but use mocks/shims instead).
* FunctionalTests: these have only access to the public contracts but no internals. They may have complicated prerequisites. These are allowed to make network, file I/O, and other types of system calls.

I'd prefer if we can add the PAL tests within either of the above categories, maintaining the naming convention tests\UnitTests and tests\FunctionalTests. 
I do like the separation you already have so we should keep the different folder (e.g. tests\UnitTests\PalTests.Windows) and the msbuild include file.

This is mainly to help easily identify the inner/outer loops so that we can quickly pick tests based on their requirements. It is also